ReactNative/js/screens/SearchResults/components/Results.jsx

```diff
@@ -106,10 +111,30 @@ function isResultAllowed({ template, provider, type }) {
   );
 }
 
+function groupBy(arr, n) {
```

Comment:
sometimes lodash can be very helpful :)
https://lodash.com/docs/4.17.11#chunk

Reply:
Agree. Will consider adding it in the future.